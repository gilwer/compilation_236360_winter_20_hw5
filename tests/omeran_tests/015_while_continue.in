void main(){
    int i;
    while (7 > i){
        print("starting while, i is");
        printi(i);
        i = i + 1;
        if (i/2 == (i-1)/2) {
            print("i is odd");
            printi(i);
            continue;
        }
    }
}