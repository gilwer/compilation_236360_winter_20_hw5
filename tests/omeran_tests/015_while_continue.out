starting while, i is
0
i is odd
1
starting while, i is
1
starting while, i is
2
i is odd
3
starting while, i is
3
starting while, i is
4
i is odd
5
starting while, i is
5
starting while, i is
6
i is odd
7
